# Empty
---
# Mesh response
string mesh_id
shape_msgs/Mesh mesh
